common/src/main/java/gov/cms/ab2d/common/dto/ContractDTO.java

```diff
@@ -17,4 +21,17 @@
 
     @EqualsAndHashCode.Exclude  // contractNumber is sufficient, breaks on Windows due sub-seconds not matching
     private String attestedOn;
+
+    public void setAttestedOn(String offsetDateTime) {
```

Comment:
Adding setters is the wrong direction especially with the allArgs constructor.  Let's change this from Data to Value, this class should really be immutable.